build_runner/lib/src/daemon/daemon_builder.dart

```diff
@@ -204,17 +206,16 @@ class BuildRunnerDaemonBuilder implements DaemonBuilder {
               // Assume we will create an outputDir.
               true,
               expectedDeletes,
-            ));
-
-    var changes = graphEvents
+            ))
         .map((data) => WatchEvent(data.type, '${data.id}'))
         .map((change) => [change]);
```

Comment:
If we skip this can we also skip the `.map((changeLists) => changeLists.expand((l) => l).toList()`?

Reply:
Oh true it does seem like that, will give it a shot